src/freenet/node/FailureTable.java

```diff
@@ -74,12 +74,12 @@ public void shouldUpdate(){
 	/** Terminate a request if there was a DNF on the same key less than 10 minutes ago.
```

Comment:
Please update the time in the documentation as well.

Reply:
I removed the value itself from the comment: that was needless duplication.

Thank you for catching it!